tests/unit/core/reducers/test_user.js

```diff
@@ -0,0 +1,42 @@
+import { logOutUser } from 'core/actions';
+import reducer, {
+  userProfileLoaded,
+} from 'core/reducers/user';
+import { createUserProfileResponse } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const { id, username } = reducer(undefined);
+      expect(id).toBe(null);
```

Comment:
I'd be fine if we just used `toEqual` everywhere because it's easier to read/rewrite/etc. I don't think we'll get bit by using it over `toBe`. @muffinresearch and I chatted about this and it's just nicer to have fewer matchers IMO.

That said if you left it here I'd be fine with it. Just a note.

Reply:
can I `sed -i s/toBe/toEqual` in another PR maybe?

Reply:
Yeah, sure, it's really no biggie. I just figured I'd mention for future patches. I think `toEqual` is a nicer matcher too; `toBe` implies more of an exact type-check not a value-check.